webapp/src/main/java/org/opentestsystem/rdw/admin/repository/impl/JdbcGroupRepository.java

```diff
@@ -57,8 +61,24 @@
         );
     }
 
+    // TODO: This implementation only updates the isDeleted field.
+    // TODO: Other fields need to be validated before they can be updated.
+    @Override
+    public void update(@NotNull final PermissionScope permissionScope, @NotNull final Group group) {
+        final int result = template.update(
+                update,
+                new MapSqlParameterSource()
+                        .addValues(getSecurityParameters(permissionScope))
+                        .addValues(getGroupParameters(group))
+        );
+
+        if(result == 0)
+            throw new NoSuchElementException();
```

Comment:
If nothing was updated, throwing an exception.  At this point, the UI shouldn't be showing the "Delete" link for groups the user cannot delete, so this shouldn't happen, but just in case.